Shared/Action/TelemetryAction.swift

```diff
@@ -48,6 +48,7 @@ enum TelemetryEventObject: String {
     case autofillOnboarding = "autofill_onboarding"
     case autofill = "autofill"
     case autofillInstructions = "autofill_instructions"
```

Comment:
maybe we can change this name to be more descriptive now that we have multiple autofill instructions?